chains/orchestrator-paras/runtime/dancebox/src/tests/common/xcm/core_buyer.rs

```diff
@@ -31,10 +31,10 @@ const ROCOCO_ED: u128 = rococo_runtime_constants::currency::EXISTENTIAL_DEPOSIT;
 const BUY_EXECUTION_COST: u128 = crate::xcm_config::XCM_BUY_EXECUTION_COST_ROCOCO;
 // Difference between BUY_EXECUTION_COST and the actual cost that depends on the weight of the XCM
 // message, gets refunded on successful execution of core buying extrinsic.
-const BUY_EXECUTION_REFUND: u128 = 3076488;
+const BUY_EXECUTION_REFUND: u128 = 24506230;
```

Comment:
The weight has decreased almost 9x?

Reply:
Maybe because of this? https://github.com/paritytech/polkadot-sdk/pull/3612

But it's not merged... @girazoki ?

Reply:
wow I did not remember I had this PR opened yet

Reply:
This is the refunded weight, so the actual cost has decreased. I copied this value from a failing test